tests/unit/core/api/test_index.js

```diff
@@ -82,7 +82,7 @@ describe(__filename, () => {
       const endpoint = 'project-ă-ă-â-â-日本語';
       mockWindow
         .expects('fetch')
-        .withArgs(sinon.match(`/api/v3/${endpoint}/`))
```

Comment:
I wonder if it would be worth making the version a `const` in this file so that if we ever need to change this again it can be changed in one place instead of in multiple tests?